examples/gnrc_border_router/Makefile

```diff
@@ -12,8 +12,8 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon b-l072z-lrwan1 bluepill calliope-mini
                              microbit msb-430 msb-430h nrf51dongle nrf6310 \
                              nucleo32-f031 nucleo32-f042 nucleo32-f303 nucleo32-l031 \
                              nucleo-f030 nucleo-f070 nucleo-f072 nucleo-f103 nucleo-f302 \
-                             nucleo-f334 nucleo-l053 nucleo-l073 opencm904 pca10000 \
-                             pca10005 spark-core stm32f0discovery telosb wsn430-v1_3b \
+                             nucleo-f334 nucleo-l053 nucleo-l073 opencm904 \
+                             spark-core stm32f0discovery telosb wsn430-v1_3b \
```

Comment:
This line cause a merge conflict in the gnrc_netif2 integration (see #7925). Is there a way to revert this for now?

Reply:
Isn't that only relevant when you actually merge?

Reply:
how about #7986